include/envoy/common/platform.h

```diff
@@ -19,13 +19,23 @@
 #include <mswsock.h>
 #include <ws2tcpip.h>
 
+#undef NTDDI_VERSION
+#define NTDDI_VERSION WDK_NTDDI_VERSION
```

Comment:
Shouldn't NTDDI_VERSION be defined to the minimum system version required (in Envoy case, whatever version afunix.h was introduced)? Looks like WDK_NTDDI_VERSION represents "latest". https://docs.microsoft.com/en-us/windows/win32/winprog/using-the-windows-headers

Reply:
see comment above. We changed the `NTDDI_VERSION` to the version that afunix was introduced